docs/reference/samples/observability.output-loki-default.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  serviceAccount:
+    name: logcollector
+  outputs:
+    - name: default-lokistack
+      type: lokiStack
+      lokiStack:
+        target:
+          name: lokistack-dev
```

If this is meant as "user documentation" instead of "developer example", we have, so far used `logging-loki` as an example name there:

```suggestion
          name: logging-loki
```